spec/services/gdynia_extra_data_ingestor_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GdyniaExtraDataIngestor', type: :service do
+  describe 'when using the call function' do
```

Comment:
The convention for describes is just to use the function's name - makes it easier to read multiple tests and 
```suggestion
  describe '#call' do
```